guides/v2.1/cloud/env/variables-intro.md

```diff
@@ -9,6 +9,7 @@ functional_areas:
 {{site.data.var.ece}} enables you to assign environment variables to override configuration options:
 
 -   [ADMIN]({{ page.baseurl }}/cloud/env/environment-vars_magento.html)—variables override project ADMIN variables
```

Comment:
variables that override ...

For parallelism, add "that" to all the descriptions --  possible just remove the em-dash and the "that"